spec/classes/puppet_server_passenger_spec.rb

```diff
@@ -6,10 +6,15 @@
     next if exclude_test_os() and exclude_test_os.include?(os)
     next if os_facts[:osfamily] == 'windows'
     context "on #{os}" do
+      let(:pre_condition) do
+        'include ::puppet'
+      end
```

Comment:
The parameters of such a sub-class needs to get set in each test then, to have that isolated.